addons/web_editor/static/src/js/editor/odoo-editor/test/spec/urlRegex.test.js

```diff
@@ -13,8 +13,8 @@ describe('urlRegex', () => {
         const match = text.match(URL_REGEX);
         chai.expect(match).to.be.equal(null);
     });
-    it('should match abc.abc.abc', () => {
-        const url = 'abc.abc.abc';
+    it('should match www.abc.abc', () => {
+        const url = 'www.abc.abc';
```

Comment:
Could you improve the unit tests instead of just replacing this one.
* `should match www.abc.abc` 
*  `should NOT match abc.abc.abc`
* `should match abc.abc.com`

Reply:
@deso-odoo 

You forgot to add a test for `should NOT match abc.abc.abc`

Reply:
@sebgeelen No No I did not forgot, I have added it [here](https://github.com/odoo/odoo/pull/135321/files#diff-3fa9b093cd4b3eb7c5faab61aecbb78e78227a77f87c61a4f98f50b47056ee79R124) by passing it in `testNotUrlRegex`. Do you want me to add it in this specific file and expect the result be equall to `null`.

Reply:
Yes please, better be verbose with tests.

Reply:
Done!